entities/dashboard/widget.json

```diff
@@ -0,0 +1,128 @@
+{
+    "type": "object",
+    "properties": {
+        "uid": {
+            "type": "string",
+            "description": "Unique identifier of the dashboardWidget object"
+        },
+        "app_code_name": {
+            "type": "string",
+            "description": "The code name of the app that owns the widget"
+        },
+        "display_name": {
+            "type": "object",
+            "description": "The widget display name in different languages"
+        },
+        "permissions": {
+            "type": "string",
+            "description": "The permissions list required to be able to use this widget, seperated bu comma"
+        },
+        "feature_flags": {
+            "type": "string",
+            "description": "The feature flags list required to be able to use this widget, seperated by comma"
+        },
+        "module": {
+            "type": "string",
+            "description": "The module that owns the widget"
+        },
+        "component_data": {
+            "type": "object",
+            "description": "a configuration object that is passed to the widget to initialize it. Can vary based on the widget class"
+        },
+        "dimensions": {
+            "min_width": {
+                "type": "integer",
+                "description": "the minimum width of the widget in grid units",
```

Comment:
Capitalize first letter (in all descriptions)
Change to (and fix all accordingly):
In grid units, the minimal width allowed for the widget to shrink to optimize the dashboard layout.